docs/how-to/share-dashboard.md

```diff
@@ -0,0 +1,63 @@
+# Share dashboards and panels
+
+When you need to share a dashboard with your team members, you can either send them a direct link to the dashboard, or render and send the dashboard as a .PNG image.
+
+## Share as direct link
+1. Go to the dashboard that you want to share.
+2. Click at the top of the dashboard to display the panel menu.
+3. Select **Share** to reveal the **Share Panel** and either:  
+   - copy and send the full URL for the dashboard
+   - toggle the **Short URL** option to generate a simple link, without all the query parameter. 
```

Comment:
There's actually still one query param (orgId). Maybe we should just mention that's a shorter link using a unique identifier